flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/catalog/CatalogFunctionImpl.java

```diff
@@ -67,6 +67,9 @@ public CatalogFunction copy() {
 
 	@Override
 	public boolean isGeneric() {
+		if (functionLanguage == FunctionLanguage.PYTHON) {
```

Comment:
What's the aim of this change?

Reply:
Currently the Python UDF is always generic and can not be loaded via `Class.forName`. So just return true if the CatalogFunction is a Python UDF.